### ISSUE TITLE
Background is black no matter what 

### ISSUE THREAD

**Issue by npahucki** [Eclair/CircleProgressBar#3]
Hi, awesome control, thanks for sharing. I am having an issue that I can't figure out looking at the source code. The View in IB has a white background color, but for some reason, the control is drawn with a black background no mater what I do. Even if I set the background color of the view in code to something else, it still shoes black. Do you have any clues for me? I"m happy to keep investigating, but if you have some ideas I'd be happy to hear them. Thank you!

```
_circleProgressBar.progressBarWidth = 15;
_circleProgressBar.startAngle = 270.0;
_circleProgressBar.hintViewBackgroundColor = [UIColor whiteColor];
_circleProgressBar.backgroundColor = [UIColor whiteColor];
_circleProgressBar.progressBarTrackColor = [UIColor appHeaderBackgroundNormalColor];
_circleProgressBar.progressBarProgressColor = [UIColor appNormalColor];
_circleProgressBar.hintTextColor = [UIColor appNormalColor];
_circleProgressBar.hintTextFont = [UIFont fontForAppWithType:Book andSize:15];
_circleProgressBar.hintHidden = NO;
```

![screenshot 2015-04-29 23 41 54](https://cloud.githubusercontent.com/assets/823555/7405474/67e6514e-eec9-11e4-9cca-a34723e42b23.png)

**Comment by loicgriffie** [comment]
Hi, Weird i had this but once i did [self.progress setBackgroundColor:[UIColor clearColor]]; in the viewDidLoad: the black color did disappeared as expected. Where do you update the properties ? loadView or viewDidLoad ? Better do it in viewDidLoad

**Comment by Eclair** [comment]
Hi @npahucki, @Appsido,

I fixed this issue in commit: 310780ae778193944fab1a6d9aafba847b528532 .
Soon will publish it to cocoapods.

Thanks,

**Comment by npahucki** [comment]
Thank you so much for the quick fix! Now if only Apple would fix XCode as fast....;-)
